server/src/main/kotlin/com/studo/campusqr/endpoints/ReportData.kt

```diff
@@ -18,80 +18,80 @@ import java.util.*
 
 //@formatter:off
 /**
-  Contact tracing by time (without seat filtering): See ContactTracingTest.kt
+Contact tracing by time (without seat filtering): See ContactTracingTest.kt
 
-                checkIn               checkOut
-                  +       infected      +
-                  +---------------------+
-                  +                     +
+checkIn               checkOut
++       infected      +
++---------------------+
++                     +
 
-                    +     a     +  +    b
-                    +-----------+  +---------->
-                    +           +  +
++     a     +  +    b
++-----------+  +---------->
++           +  +
 
-             +    c    +           +    d    +
-             +---------+           +---------+
-             +         +           +         +
++    c    +           +    d    +
++---------+           +---------+
++         +           +         +
 
-     +   e   +                               +    f    +
-     +-------+                               +---------+
-     +       +                               +         +
++   e   +                               +    f    +
++-------+                               +---------+
++       +                               +         +
 
-              +               g               +
-              +-------------------------------+
-              +                               +
-  |----------------------------------------------------> time
++               g               +
++-------------------------------+
++                               +
+|----------------------------------------------------> time
```

Comment:
pls revert formatting, not sure why "//@formatter:off" doesn't work. Probably we just need to move the big comment here to a endpoints/Readme file and reference the documentation here.

Reply:
It doesn't work because I haven't ported the "Honor formatter markers" task to all projects ^^  But it's on my todo list.